MODULE.bazel

```diff
@@ -16,19 +16,9 @@ bazel_dep(name = "libpfm", version = "4.11.0")
 python = use_extension("@rules_python//python/extensions:python.bzl", "python")
 python.toolchain(python_version = "3.9")
 
-# Extract the interpreter from the hermetic toolchain above, so we can use that
-# instead of the system interpreter for the pip compiplation step below.
-interpreter = use_extension("@rules_python//python/extensions:interpreter.bzl", "interpreter")
-interpreter.install(
-    name = "interpreter",
-    python_name = "python_3_9",
-)
-use_repo(interpreter, "interpreter")
-
 pip = use_extension("@rules_python//python/extensions:pip.bzl", "pip")
 pip.parse(
-    name="tools_pip_deps",
-    incompatible_generate_aliases = True,
-    python_interpreter_target="@interpreter//:python",
+    hub_name="tools_pip_deps",
+    python_version = "3.9",
```

Comment:
It's possible this line shouldn't be here; am in discussion with the rules_python authors around this design decision as this is problematic when the main repo declares a different default python_version.

Reply:
i see.. so this can be different to the `python.toolchain` declared above?

Reply:
See https://github.com/bazelbuild/rules_python/issues/1320